src/core/utils.js

```diff
@@ -81,9 +82,12 @@ export function fromJSOrdered(js) {
     return Im.Seq(js).map(fromJSOrdered).toList()
   }
   if (js.entries) {
```

Comment:
I'd reduce the cyclomatic complexity by order of 1 into otherwise PR looks legit.
```js
if (isFunction(js.entries)) {
    // handle multipart/form-data
    const objWithHashedKeys = createObjWithHashedKeys(js)
    return Im.OrderedMap(objWithHashedKeys).map(fromJSOrdered)
} 
if (js.entries && !isFunction(js.entries)) {
   return Im.OrderedMap(js.entries).map(fromJSOrdered)
}
```